packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3997,11 +3997,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       return (
         <div style={wrapperStyle}>
           <Tooltip options={popperOptions} ref={this.handleTooltipRef}>
-            <div className="link-hover-tooltip">
-              {linkHoverDisplayValue} - Click once to follow.
-              <br />
-              Click and hold to select this cell.
-            </div>
+            <div className="link-hover-tooltip">{hoverDisplayValue}</div>
```

Comment:
I wonder if the flakiness is the tooltip appearing as we're moving the mouse around. Maybe moving the mouse somewhere outside of the grid instead of waiting inside the grid before taking a snapshot.